recipes/namd/3.0b6/gh200/environments.yaml

```diff
@@ -0,0 +1,15 @@
+namd:
+  compiler:
+  - toolchain: gcc
+    spec: gcc@12.3
+  unify: true
+  specs:
+  - cuda@12
+  - fftw@3.3.10 +openmp ~mpi
+  - charmpp@7 backend=multicore +production +tracing
+  - namd@3.0b6 +cuda cuda_arch=90 +single_node_gpu
+  views:
+    namd-dev:
```

Comment:
How about `develop` as a name for the development environment?
It is a bit more generic - other environments can provide a `develop` view?